interface/new/new_comprehensive.php

```diff
@@ -98,6 +99,16 @@ function getSearchClass($data_type) {
 <?php include_once("{$GLOBALS['srcdir']}/options.js.php"); ?>
 <link rel="stylesheet" type="text/css" href="../../library/js/fancybox/jquery.fancybox-1.2.6.css" media="screen" />
 
+<?php
+    //this will hold the javascript submit me param new_validate;
+    $new_validate = $GLOBALS['new_validate'] ? 1 : 0;
```

Comment:
Above variable is not used anywhere so can go. And would change the comment to a quick 1 liner on the zend hook.